apps/demos/Demos/Splitter/Overview/Angular/app/app.component.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  NgModule, ChangeDetectionStrategy, Component, ViewChild, enableProdMode, Input,
+} from '@angular/core';
+import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { DxSplitterModule } from 'devextreme-angular';
+
+if (!document.location.host.includes('localhost')) {
+  enableProdMode();
+}
+
+@Component({
+  selector: 'demo-app',
+  templateUrl: 'app/app.component.html',
+  styleUrls: ['app/app.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppComponent {
+  dimensionOptions = new Set(['size', 'minSize', 'maxSize']);
+
+  @Input() paneContentTemplates: any[];
+
+  renderedContentTemplates: any[];
+
+  constructor() {
+    this.paneContentTemplates = [
+      { name: 'Left Pane' },
+      { name: 'Central Pane' },
+      { name: 'Right Pane' },
+      { name: 'Nested Left Pane' },
+      { name: 'Nested Central Pane' },
+      { name: 'Nested Right Pane' },
+    ];
+  }
+
+  ngOnInit(): void {
+    this.renderedContentTemplates = JSON.parse(JSON.stringify(this.paneContentTemplates));
```

Comment:
Whats is this for?

Reply:
It's the first attempt to prevent size, maxsize value updating in Panes. I used ChangeDetections for this purpose and forgot to remove this.

Reply:
https://github.com/DevExpress/DevExtreme/pull/26972/commits/3231da54f808636759096a70ef290df3b4b763ff